distribution/solr-distro/pom.xml

```diff
@@ -99,12 +99,13 @@
                         </goals>
                         <configuration>
                             <url>
-                                http://artifacts.codice.org/content/repositories/thirdparty/org/apache/archive/solr/${solr.version}/solr-${solr.version}.zip
```

Comment:
:question: Why 7.4 instead of solr.version which is currently 7.7.2?

Reply:
I thought it was time for a downgrade :-) I'll change it to 7.7.2

Reply:
Actually meant to keep the property ${solr.version} there